azure-devops/azext_devops/dev/repos/policy.py

```diff
@@ -0,0 +1,371 @@
+        policy_client = get_policy_client(organization)
+        return policy_client.get_policy_configurations(project=project)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def get_policy(id, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
+    """
+    :param id: ID of the policy.
+    :type id: int
+    :param organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: Automatically detect organization. Default is "on".
+    :type detect: str
+    :rtype: :class:`<PolicyConfiguration> <policy.v4_0.models.PolicyConfiguration>`
+    """
+    try:
+        organization, project = resolve_instance_and_project(
+            detect=detect, organization=organization, project=project)
+        policy_client = get_policy_client(organization)
+        return policy_client.get_policy_configuration(project=project, configuration_id=id)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+
+def delete_policy(id, organization=None, project=None, detect=None):  # pylint: disable=redefined-builtin
+    """
+    :param id: ID of the policy.
+    :type id: int
+    :param organization: Azure Devops organization URL. Example: https://dev.azure.com/MyOrganizationName/
+    :type organization: str
+    :param project: Name or ID of the project.
+    :type project: str
+    :param detect: Automatically detect organization. Default is "on".
+    :type detect: str
+    """
+    try:
+        organization, project = resolve_instance_and_project(
+            detect=detect, organization=organization, project=project)
+        policy_client = get_policy_client(organization)
+        return policy_client.delete_policy_configuration(project=project, configuration_id=id)
+    except VstsServiceError as ex:
+        raise CLIError(ex)
+
+# pylint: disable=too-many-locals
+def create_policy(repository_id, branch,
+                  isBlocking=False, isEnabled=False,
+                  policy_type=None,
+                  minimumApproverCount=None, creatorVoteCounts=None, allowDownvotes=None, resetOnSourcePush=None,
+                  useSquashMerge=None,
+                  buildDefinitionId=None, queueOnSourceUpdateOnly=None, manualQueueOnly=None, displayName=None, validDuration=None,
+                  maximumGitBlobSizeInBytes=None, useUncompressedSize=None,
+                  optionalReviewerIds=None, requiredReviewerIds=None, message=None,
+                  organization=None, project=None, detect=None):
+    """
+    :param repository_id: Id (UUID) of the repository on which to apply the policy
+    :type repository_id: string
+    :param branch: Branch on which this policy should be applied
+    :type branch: string
+    :param isBlocking: Tells if the policy should be blocking or not
```

Comment:
Whether the policy should be blocking or not. Default value is false. Accepted values are true and false.